site/faq.md

```diff
@@ -99,6 +99,13 @@ explanation.
 Flux needs a deploy key to be allowed to push to the version control
 system in order to read and update the manifests.
 
+
+### What is auto-matically synced back to git by Flux?
+
+Nothing is auto-matically sent from the cluster to git! However, some Flux commands and API
+calls will update the git repo so as to reflect the current state of the cluster or the new state
```

Comment:
The `fluxctl` commands don't reflect the current state of the cluster so much as they change the desired state of the cluster, as expressed in the manifests in git. It's not really a qualification -- just a related bit of information. You could mention automated updates here too.